hw/ip/usbdev/data/usbdev_testplan.hjson

```diff
@@ -511,12 +492,52 @@
       stage: V2
       tests: []
     }
+    {
+      name: enable
+      desc: '''
+            Verify that DUT connects to the USB when enable is set.
+
+            - set USB_CTRL_enable pin high.
+            - Make sure 'phy_config.pinflip' isn't set.
+            - Verify that the 'usb_dp_pullup_o' pin goes high.
+            '''
+      stage: V2
+      tests: ["usbdev_enable"]
+    }
+    {
+      name: resume_link_active
+      desc: '''
+            Verify that when Write a 1 to this bit to instruct usbdev to jump to the LinkResuming state
+            then write will only have an effect when the device is in the LinkPowered state.
+
+            - Perform no operation in J state for more than 3 ms.
+            - Read the link state from usbstate register to see if it is in powered state and then send the
+              resume signal (K state) for 20 ms and set the usbctrl.resume_link_active.
+            - Verify that the resume_link_active interrupt goes high and device will be resumed.
+            '''
+      stage: V2
+      tests: []
+    }
+    {
+      name: device_address
+      desc: '''
+            Verify that the device ignores all packets intended for other addresses.
+
+            - Assigned device address to (usbctrl[22:16]) and send packets to this address.
+            - Send packets to other addresses.
+            - Verify that the device responds to all packets sent by the DV environment (USB side)
+              that are sent to its assigned address.
```

Comment:
As we're looking at this text anyway... This could be a lot shorter! Probably "Verify that the device responds to all packets sent to its assigned address." ?